src/TesApi.Tests/StartupTests.cs

```diff
@@ -141,6 +141,34 @@ public void ConfigureServices_TerraOptionsAreNotConfigured_DefaultActionIdentity
             Assert.IsInstanceOfType(actionIdentityProvider, typeof(DefaultActionIdentityProvider));
         }
 
+        [TestMethod]
```

Comment:
I'm not sure if this was a bad merge or what, but it appears that the only changes in this file are these two methods are duplicated. Just revert this file, please.